src/sap/fhir/model/r4/FHIRListBinding.js

```diff
@@ -161,7 +161,8 @@ sap.ui.define([
 				this.bPendingRequest = false;
 				this.bInitial = false;
 				var oBindingInfo = this.oModel.getBindingInfo(this.sPath, this.oContext, this.bUnique);
-				var sStrucDefUrl = this.oModel.getProperty(oBindingInfo.getResourcePath()).meta.profile[0];
+				var oResource = this.oModel.getProperty(oBindingInfo.getResourcePath());
```

Comment:
oResource = this.oModel.getProperty(oBindingInfo.getResourcePath()) || {resourceType: oBindingInfo.getResourceType()};

Reply:
i tweaked it a bit since the resource was not undefined and there are cases when resourcetype property is not present but some other property was there
var oResource = this.oModel.getProperty(oBindingInfo.getResourcePath()) || {};
oResource.resourceType = oResource.resourceType || oBindingInfo.getResourceType();